features/jupyter.feature

```diff
@@ -4,6 +4,10 @@ Feature: Jupyter targets in new project
     Given I have prepared a config file
     And I have run a non-interactive kedro new with starter "default"
 
+  Scenario: Execute jupyter-init target
```

Comment:
```suggestion
  Scenario: Execute jupyter init target
```

Reply:
Nitpick, but the other tests in this file are written in a similar weird way and should probably be changed `jupyter-notebook` ➡️ `jupyter notebook` etc.